### ISSUE TITLE
OverrideActionBar

### ISSUE THREAD

**Issue by eightflat** [brittyazel/EnhancedRaidFrames#59]
I get this error via BugGrabber/BugSack pretty regularly. Does it look like anything on the side of EnhancedRaidFrames, or is it more likely a bad interaction with another addon? Thanks for all your work!

```
13x [ADDON_ACTION_BLOCKED] AddOn 'EnhancedRaidFrames' tried to call the protected function 'OverrideActionBarButton6:Show()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `Show'
[string "@FrameXML\ActionButton.lua"]:368: in function `ActionButton_Update'
[string "@FrameXML\ActionButton.lua"]:829: in function `ActionButton_OnEvent'
[string "@FrameXML\ActionButton.lua"]:257: in function <FrameXML\ActionButton.lua:249>
```

**Comment by brittyazel** [comment]
Hrm, yes that is definitely not something that ERF is doing... so that's quite odd. Do you have any idea which addon might be having the poor interaction? Can you try enabling them one at a time until the error starts occuring?

**Comment by eightflat** [comment]
I will try to narrow it down. Thanks for the sanity check to work off of :)

**Comment by zeheras** [comment]
Getting this as well, was just trying out settings in a proving grounds and got the following in bugsack

`1x [ADDON_ACTION_BLOCKED] AddOn 'EnhancedRaidFrames' tried to call the protected function 'OverrideActionBarButton1:Show()'.
[string "@!BugGrabber\BugGrabber.lua"]:519: in function <!BugGrabber\BugGrabber.lua:519>
[string "=[C]"]: in function `Show'
[string "@FrameXML\ActionButton.lua"]:368: in function `ActionButton_Update'
[string "@FrameXML\ActionButton.lua"]:829: in function `ActionButton_OnEvent'
[string "@FrameXML\ActionButton.lua"]:257: in function <FrameXML\ActionButton.lua:249>

Locals:
Skipped (In Encounter)`

**Comment by brittyazel** [comment]
This isn't directly caused by ERF. It seems like the fact that this issue is being attributed to ERF is an artifact. I can assure you that ERF is doing nothing to affect your OverrideActionButtons :-)

The only other thing I can think of is that there is a taint issue that happens sometimes with party members from different servers. I'm not sure if this is the same issue or not, but it's worth investigating. There are some client bugs on Blizzard's end that I still need to work around :-/

**Comment by eightflat** [comment]
Just wanted to follow-up here, with the hope it helps anyone that may come across the same issue. I tried disabling add-ons one by one, and no matter how much I didn't want to accept it, the only way I could replicate the error was with ERF enabled. I decided to do a clean re-install (deleting my saved variables for ERF), and now I don't get the error!

I've been using ERF for a long time and have been using some alpha and beta versions, so I'm guessing it has something to do with layering all those versions over each other?

At any rate, I am so happy to be able to use ERF again. I tried going back to Grid and you just can't beat the simplicity and reliability of the default raid frames with ERF! :D
